Sources/Fuzzilli/Engines/FuzzEngine.swift

```diff
@@ -20,15 +20,68 @@ public protocol FuzzEngine: ComponentBase {
 }
 
 extension FuzzEngine {
+    private func instrumentForDifferentialTesting(_ program: Program) -> Program {
```

Comment:
Could you add a short comment describing what this function is trying to achieve? From what I can tell it does
1. Create a number of global variables that will be used to store "regular" differential values
2. Interleave random `fuzzilli_hash` calls for local variables
3. Randomly assign local variables to the global variables created in step 1
4. Hash all of the variables created in step 1